lib/manageiq-providers-ibm_cloud.rb

```diff
@@ -1,2 +1,3 @@
 require "manageiq/providers/ibm_cloud/engine"
 require "manageiq/providers/ibm_cloud/version"
+require_relative "manageiq/providers/ibm_cloud/cloud_tool"
```

Comment:
@agrare, as you found in https://github.com/ManageIQ/manageiq-rpm_build/pull/137, it seems this line breaks the build as it eager loads this too early in the cycle.

Reply:
https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/145